### ISSUE TITLE
posts on projects page should be pulled automatically, not hardcoded

### ISSUE THREAD

**Issue by whoburg** [aeroa/aeroa.github.io#17]
The posts on the projects page are currently hardcoded; they should instead be pulled from the relevant subdirectory of `_posts` in a loop, like is done on the homepage.

**Comment by bqpd** [comment]
This can't be done without adding an off-github build step, sadly.

**Comment by 1ozturkbe** [comment]
Yeah, unfortunate... I have been looking a little into how to tag posts with categories; it may be the solution. Look into [this](https://blog.webjeda.com/jekyll-categories/) link. I haven't had time to dig into it, but it seems like there is a method at least to easily sorting posts by category, and create a new page for each category.
